cmd/ipfs-pinset-migration/main.go

```diff
@@ -22,11 +23,12 @@ func main() {
 		flag.PrintDefaults()
 		os.Exit(1)
 	}
-	migrate.Migrate(bucket, pinsPrefix, ipfsApiUrl)
+	migrate.Migrate(bucket, pinsPrefix, ipfsApiUrl, statusPath)
 }
 
 func init() {
 	flag.StringVarP(&bucket, "bucket", "b", "", "S3 bucket name")
 	flag.StringVarP(&pinsPrefix, "prefix", "p", "", "S3 bucket prefix for pins")
 	flag.StringVarP(&ipfsApiUrl, "ipfs", "i", "localhost:5001", "IPFS API url")
+	flag.StringVarP(&statusPath, "status", "s", "/tmp", "path for status files")
```

Comment:
```suggestion
	flag.StringVarP(&logPath, "logPath", "l", "/tmp", "Path to store log files")
```

Or `resultsPath`. I feel like status is a confusing cli flag for this

Reply:
`logPath` is cool 👍🏼